vunit/test/acceptance/test_external_run_scripts.py

```diff
@@ -115,6 +116,13 @@ def test_vhdl_array_axis_vcs_example_project(self):
     def test_vhdl_axi_dma_example_project(self):
         self.check(join(ROOT, "examples", "vhdl", "axi_dma", "run.py"))
 
+    @unittest.skipIf(
+        GHDLInterface.supports_vhpi(),

```

Comment:
I do not understand why a GHDLInterface method is called here.

Reply:
It is used to check if the simulator supports VHPI before trying to run the tests. I think that it might be possible to replace it with `SimulatorInterface`, even if `supports_vhpi` is not implemented in other simulator interfaces. But I am not sure about it.

Reply:
The acceptance tests are run with different environment variables to select different simulators. A call like `GHDLInterface.supports_vhpi()` would always return the same thing regardless of which simulator that was selected and is supposed to run the test. The easiest way is just to currently only run the test if GHDL is selected by having a `simulator_is("ghdl")`. 

A more future proof way would be to add a new function to vunit/test/common.py called `simulator_check` that takes a lamba function which will take the simulator class of the selected simulator and return True or False based on if the test should run or not. For example:
```python
@unittest.skipIf(simulator_check(lambda simclass: not simclass.supports_vhpi())
```

Reply:
Thanks for the insight. I will try to go with the lambda function.

Reply:
> The easiest way is just to currently only run the test if GHDL is selected by having a `simulator_is("ghdl")`.

I think this would be dirtier than the lambda function, because not all GHDL backends support VHPI. So we would need to evaluate two conditions.

Reply:
> Thanks for the insight. I will try to go with the lambda function.

This is implemented now.